README.md

```diff
@@ -1,25 +1,24 @@
-# Clearhaus 3D-Secure v2 API Documentation #
+# Clearhaus 3-D Secure v2 API Documentation
 
-## Status ##
+## Status
```

Comment:
I like appending hashes to make it readable in the textual output, but I don't care too much :)